locations/templates/quarkus-application/manifests/helm/deploy/templates/_helpers.tpl

```diff
@@ -65,6 +65,7 @@ Selector labels
 {{- define "quarkus-template.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "quarkus-template.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
+app.quarkus.io/quarkus-version: {{ .Values.app.quarkusVersion }}
```

Comment:
Why did you added the Quarkus version as new selectorLabel ? If you add it here logically it should also be part of the labels too ?

Reply:
I don't know exactly why I added to the selectorLabels. Anyway these labels will be also part of the labels too, they are included by [this line](https://github.com/q-shift/backstage-playground/blob/f67696a46771ef894051f4261773413accc256d8/locations/templates/quarkus-application/manifests/helm/deploy/templates/_helpers.tpl#L55), right @iocanel ?

Reply:
Ok if they are also included within the labels. The selectorLabels are used to find a service, etc